include/mbedtls/ssl_ciphersuites.h

```diff
@@ -168,6 +168,61 @@ extern "C" {
 #define MBEDTLS_TLS_ECDHE_PSK_WITH_NULL_SHA256           0xC03A /**< Weak! No SSL3! */
 #define MBEDTLS_TLS_ECDHE_PSK_WITH_NULL_SHA384           0xC03B /**< Weak! No SSL3! */
 
+#define MBEDTLS_TLS_RSA_WITH_ARIA_128_CBC_SHA256          0xC03C
```

Comment:
Please add comments indicating version restrictions. As usual the RFC is ambiguous regarding the minimum version. On a technical level, the EC suites need at least TLS 1.0 and the GCM/CCM suites need TLS 1.2, and it is my impression that most implementers (including us so far), allow new ciphersuites to be used with older versions as far as technically possible, as long as the RFC doesn't explicitly prohibit it.

OTOH, I could hear arguments for only adding new suites only to TLS 1.2, but as this would be a change compared to what we previously did, it would need to be deliberate and justified.